pylint/extensions/docparams.py

```diff
@@ -309,14 +309,24 @@ def visit_raise(self, node: nodes.Raise) -> None:
         doc = utils.docstringify(func_node.doc, self.config.default_docstring_type)
         if not doc.is_valid():
             if doc.doc:
-                self._handle_no_raise_doc(expected_excs, func_node)
+                missing = {exc.name for exc in expected_excs}
+                self._handle_no_raise_doc(missing, func_node)
             return
 
         found_excs_full_names = doc.exceptions()
-
```

Comment:
Could you re-add this? These white lines make the code a little more readable imo.